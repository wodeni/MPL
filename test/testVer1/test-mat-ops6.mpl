int main()
{
  Mat<float>[2][2] m;
  float i;
  i = 2.5;
  m = [1,2;3,4];
  printm(m*i);
  return 0;
}