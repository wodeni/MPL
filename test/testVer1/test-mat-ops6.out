[
2.5, 5.0;
7.5, 10.0;
]